docs/Meadow/Meadow.OS/Cellular/index.md

```diff
@@ -174,11 +192,11 @@ void CellAdapter_NetworkConnected(INetworkAdapter networkAdapter, INetworkAdapte
 }
 ```
 
-> **Notes**: Before using the mentioned properties, ensure a successful connection has been established. The `Csq` property returns a static value (0-31) representing the signal quality obtained on the connection.
+> **Notes**: Before using the mentioned properties, ensure a successful connection has been established. The `CSQ` property returns a static value (0-31) representing the signal quality obtained on the connection.
```

Comment:
Just to let you know that the name of the property is `Csq` in fact, it's why I let it instead of `CSQ`, but if you think that using `CSQ` fits better, no problem from my side.

Reply:
You're totally right! reverted!